packages/ui/src/components/UrbitSigil.native.tsx

```diff
@@ -1,13 +1,62 @@
 // Note: we must import sigil from @urbit/sigil-js/dist/core
```

Comment:
Can we extract everything these two files have in common to a `UrbitSigil.base.tsx`? There's a lot of overlap.